core/domain/state_domain.py

```diff
@@ -1007,8 +1004,15 @@ def get_all_outcomes(self) -> List[Outcome]:
 
     def _validate_continue_interaction(self) -> None:
         """Validates Continue interaction."""
-        text_value = (
-            self.customization_args['buttonText'].value.unicode_str)
+        # Here we use cast because we are narrowing the down the type from
```

Comment:
narrowing the down --> narrowing down

Ditto elsewhere.

Reply:
comment is updated